src/frequenz/client/electricity_trading/_client.py

```diff
@@ -49,6 +50,32 @@ class _Sentinel:
 NO_VALUE = _Sentinel()
 
 
+def validate_decimal_places(value: Decimal, decimal_places: int, name: str) -> None:
```

Comment:
I wouldn't make this function take a name, the caller should add this information when catching the exception, and if the exception is not caught, then the backtrace will give you this information.

Reply:
On a second thought, if this function is not exported and it is just an internal utility function, then I think it is fine to keep the name.